modules/spacelift/variables.tf

```diff
@@ -1,11 +1,23 @@
-variable "runner_image" {
+variable "region" {
   type        = string
-  description = "Full address & tag of the Spacelift runner image (e.g. on ecr)"
+  description = "AWS Region"
+}
+
+variable "runner_image" {
+  type        = any
+  description = "Full address & tag of the Spacelift runner image (e.g. on ECR)"
 }
 
 variable "worker_pool_id" {
-  type        = string
-  description = "Alpha-numberic worker poold ID of the Spacelift worker pool to use"
+  type        = any
```

Comment:
Let's retain the original types

We had to switch to `any` in a previous client but no longer need to